Automattic/wp-calypso
# Update Cloudflare/GA analytics tools to latest design/behavior

#### Changes proposed in this Pull Request

* This PR refactors the GA/Cloudflare Analytics cards on Marketing -> Traffic to match the designs listed in pbAPfg-1fn-p2. It also adds logic by which GA will be disabled when a new Cloudflare tracking ID gets set, and vice versa.

#### Testing instructions

* Check out this diff to your local Calypso environment.
* Check out D56767-code to your sandbox.
* Navigate to Tools->Marketing->Traffic on a free site account.
* Verify that the Cloudflare and Google Analytics tool options are listed as requiring a premium or higher account.

![image](https://user-images.githubusercontent.com/13437011/107697627-155a1580-6c79-11eb-91b4-9e4161e3d987.png)

* Navigate to Tools->Marketing->Traffic on a premium account.
*  Verify that you can enable the Cloudflare analytics feature.
* Paste the following snippet into the text field:
```

<script defer 
    src='https://static.cloudflareinsights.com/beacon.min.js' 
    data-cf-beacon='{"token": "716d4d275266427cb78934b7919e73e6"}'>
</script>

```
* Verify that when tabbing out of the field (or otherwise blurring it) this snippet is condensed to only the token value (i.e. `716d4d275266427cb78934b7919e73e6`)
* After saving this update, verify that, when visiting the site, you can see a Cloudflare analytics block in the footer code:
<img width="1462" alt="Screen Shot 2021-02-03 at 2 19 47 PM" src="https://user-images.githubusercontent.com/13437011/106804409-f9c48e80-662a-11eb-8566-17bd2bdc83f4.png">
* Return to Tools->Marketing->Traffic. Verify that you can enable the Google Analytics plugin as well.



Related to 428-gh-Automattic/dotcom-manage
